src/main/java/uk/gov/di/handlers/AuthCallbackHandler.java

```diff
@@ -16,7 +16,7 @@
 
 public class AuthCallbackHandler implements Route {
 
-    private static final Logger LOG = LoggerFactory.getLogger(AuthCallbackHandler.class);
+    private static final Logger LOG = LogManager.getLogger(AuthCallbackHandler.class);
```

Comment:
Can we keep the slf4j binding and wire it up to use log4j under the hood? https://www.slf4j.org/manual.html#swapping

I'm worried we'll lose framework logs like this

Reply:
Done - yes we were missing some. It's a slightly different binding to the one you shared as I think that's just for log4j 1.x. I've added the changes as a new commit so hopefully it's easier to rereview, but happy to fix up the commit history for when it gets merged if you like

Reply:
LGTM, please squash away